recipes/tttrlib/build.sh

```diff
@@ -0,0 +1,26 @@
+mkdir b2 && cd b2
+
+if [[ "${target_platform}" == osx-* ]]; then
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+  CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake \
+ -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+ -DCMAKE_PREFIX_PATH="$PREFIX" \
+ -DBUILD_PYTHON_INTERFACE=ON \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_LIBRARY_OUTPUT_DIRECTORY="$SP_DIR" \
+ -DCMAKE_SWIG_OUTDIR="$SP_DIR" \
+ -DBUILD_LIBRARY=ON \
+ -DPYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())')\
```

Comment:
conda-build sets a `PY_VER` environment variable which may be easier to use.

https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html